pdfannots/__init__.py

```diff
@@ -71,6 +71,19 @@ def _mkannotation(
         # decode as string, normalise line endings, replace special characters
         contents = cleanup_text(pdfminer.utils.decode_text(contents))
 
+    color = pa.get('C')
+    hex_color = ""
+    if color is not None:
+        # color is an array of 3 floats in the range 0.0 to 1.0
+        if len(color) != 3:
```

Comment:
By 'array' do you mean list?

It's better to check the entire type, e.g.:
```
if not (isinstance(color, list) and len(color) == 3 and all(isinstance(e, float) for e in color)):
```
... or wrap the conversion code in a suitable try/catch block.

Reply:
Also check that all the floats are >= 0 and < 1.0